src/Lucene.Net.CodeAnalysis.Dev/AnalyzerReleases.Unshipped.md

```diff
@@ -1,7 +1,10 @@
 ### New Rules
 
- Rule ID       | Category | Severity | Notes
----------------|----------|----------|-----------------------------------------------------------------------------------------------------------------------------------------------------------
- LuceneDev1007 | Design   | Warning  | Generic Dictionary<TKey, TValue> indexer should not be used to retrieve values because it may throw KeyNotFoundException (value type value)
- LuceneDev1008 | Design   | Warning  | Generic Dictionary<TKey, TValue> indexer should not be used to retrieve values because it may throw KeyNotFoundException (reference type value)
- LuceneDev6000 | Usage    | Info     | IDictionary indexer may be used to retrieve values, but must be checked for null before using the value
+| Rule ID       | Category | Severity | Notes                                                                                                                                           |
```

Comment:
Please change the table format back to how it was originally. The Analyzer for checking the rule format is fragile and is now showing warnings.

<img width="1055" height="535" alt="image" src="https://github.com/user-attachments/assets/7c1d16b7-8834-4b2d-905e-13398254a183" />